packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -298,12 +305,21 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         // update protocol fees if origin fee amount exists
         if (transaction.originFeeAmount > 0) protocolFees[transaction.originToken] += transaction.originFeeAmount;
 
-        // transfer origin collateral less fee to specified address
-        address token = transaction.originToken;
-        uint256 amount = transaction.originAmount;
```

Comment:
Same here re: caching the values from memory onto the stack.

Reply:
i thought that was already happening efficiently by the transaction struct being in mem (ie: anything accessed within it would be as efficient as repeating them into dedicated vars -- and more concise code)

will defer to your expertise & switch these back to dedicated vars